platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/reconnect/LearnerPushVirtualTreeView.java

```diff
@@ -253,13 +245,13 @@ public Long deserializeInternal(final SerializableDataInputStream in) throws IOE
         if (node == ROOT_PATH) {
             // We send the first and last leaf path when reconnecting because we don't have access
             // to this information in the virtual root node at this point in the flow, even though
-            // the info has already been sent and resides in the VirtualMapState that is a sibling
+            // the info has already been sent and resides in the ExternalVirtualMapState that is a sibling
```

Comment:
If `ExternalVirtualMapState` exists only for backward compatibility, then it shouldn't be mentioned in actual reconnect, correct? @imalygin 
I also removed `ExternalVirtualMapState` mentions (class registration) from reconnect tests and everything passed.

Reply:
@imalygin can you please update this doc then? (along with removing `ExternalVirtualMapState` from tests).